internal/command/docker.go

```diff
@@ -137,7 +203,33 @@ func goBuild(ctx Context) error {
 
 	// add ldflags to command, if any
 	if len(ldflags) > 0 {
-		args = append(args, "-ldflags", fmt.Sprintf("'%s'", strings.Join(ldflags, " ")))
+		flags := make([]string, len(ldflags))
```

Comment:
It seems passing multiple flags does not work:
```fyne-cross  linux -ldflags="-X=main.Name=Podman -X=main.Title=Test" --debug```
```
/usr/bin/docker run --rm -t -w /app -v /code/lucor/test:/app:z -v /home/lucor/.cache/fyne-cross:/go:z -e CGO_ENABLED=1 -e GOCACHE=/go/go-build -e GOOS=linux -e GOARCH=amd64 -e CC=gcc -e GOFLAGS= -ldflags=-w -ldflags=-s -e fyne_uid=1000 docker.io/fyneio/fyne-cross:1.1-base go build -ldflags=-X=-X=main.Title=Test -w -s -o /app/fyne-cross/bin/linux-amd64/test -v .
flag provided but not defined: -w
usage: go build [-o output] [build flags] [packages]
Run 'go help build' for details.
[✗] exit status 2
```
If you agree, I'd add some tests on this part of logic to ensure we won't break any previous command relying on the fyne-cross 'ldflags` or GOFLAGS env variable.

Reply:
There are many problems that come along with the modifications, I'm sorry about this.
I will try to fix this, I really don't understand what happens when env vars are passed to Docker and Podman, sometimes it takes the entire string as one argument, sometimes not...

Reply:
OK, the problem is that:
```
-ldflags="-X=A=B -X=C=D"
```

is parsed as "one" ldflags, there is no split made.
I will try another way to treat this.

Reply:
Fixed in https://github.com/fyne-io/fyne-cross/pull/42/commits/ad1b9d7f6b17dbed9f8380a4b398763c606a30ae but I don't understand how to add a test for this